tests/test_kldiv.py

```diff
@@ -145,5 +145,15 @@ def test_oned(session_tf, white, mu, sqrt, K_batch):
     np.testing.assert_allclose(kl.eval(), kl_tf.eval())
 
 
+def test_unknown_size_inputs(session_tf):
+    mu_ph = tf.placeholder(settings.float_type, [None, None])
+    sqrt_ph = tf.placeholder(settings.float_type, [None, None, None])
+    mu = np.ones([1, 4], dtype=settings.float_type)
```

Comment:
np.ones => tf.ones

Reply:
Does that work, even for passing them to the placeholders? (I tested and it seems that it doesn't). The point of having np.ones is to be able to have unknown-size tensors later.